python/tests/graph/test_pieces.py

```diff
@@ -169,6 +169,6 @@ def test_pipeline(self):
                 # tfx.write_visualization_html(issn.graph,
                 # NamedTemporaryFile(prefix="gdef", suffix=".html").name)
 
-            self.assertTrue(np.all(preds_tgt == preds_ref))
+            np.testing.assert_array_almost_equal(preds_tgt, preds_ref, decimal=5)
```

Comment:
can we use `featurizerCompareDigitsExact` setting centrally, if we want to control this?